lib/spack/spack/package.py

```diff
@@ -1926,23 +1926,31 @@ def run_test(self, exe, options, expected, status):
             exe (str): the name of the executable
             options (list of str): list of options to pass to the runner
             expected (list of str): list of expected output strings
-            status (int or None): the expected process status if int or None
-                if the test is expected to succeed
+            status (int or list of int): possible passing status values with
+                0 and None meaning the test should succeed
         """
-        result = 'fail with status {0}'.format(status) if status else 'succeed'
-        tty.debug('test: {0}: expect to {1}'.format(exe, result))
+        case = '' if isinstance(status, int) else 'in '
+        tty.msg('test: {0}: expect command status {1}{2}'
```

Comment:
Can you leave this as a debug message? I think it's too verbose to be non-conditional.

Reply:
Oops.  Definitely!